### DISCUSSION TITLE
Pas de connexion mobile vers PC

### DISCUSSION THREAD

**Discussion by Narbe77DEG** [localsend/localsend#1806] [L-Français]
Bonjour,
Envoi impossible d'un fichier de mon téléphone vers le PC. Le PC n'est pas visible depuis l'application du mobile.
Dans l'autre sens, le PC trouve bien le mobile.
Une petite idée de dépannage ?
Merci

**Comment by fiorens** [comment]
Bonjour,

Il faut que le smartphone et le pc soient sur le mêle réseau. Par exemple le réseau Wi-Fi de votre box.
Bien à vous,
